lib/qiniu/version.rb

```diff
@@ -4,7 +4,7 @@ module Qiniu
     module Version
       MAJOR = 6
       MINOR = 9
-      PATCH = 0
+      PATCH = 1
```

Comment:
v6.9.0 hadn't released yet and we can merge more changes, do the release with another pr ?